worker/src/test/java/gov/cms/ab2d/worker/processor/BundleUtils.java

```diff
@@ -1,18 +1,30 @@
 package gov.cms.ab2d.worker.processor;
 
 import gov.cms.ab2d.worker.adapter.bluebutton.ContractBeneficiaries;
+import gov.cms.ab2d.worker.processor.domainmodel.Identifiers;
 import org.hl7.fhir.dstu3.model.*;
 
-import java.util.Calendar;
-import java.util.Collection;
-import java.util.List;
+import java.util.*;
 import java.util.stream.Collectors;
 
 public class BundleUtils {
 
     public static final String BENEFICIARY_ID = "https://bluebutton.cms.gov/resources/variables/bene_id";
     public static final String MBI_ID = "http://hl7.org/fhir/sid/us-mbi";
 
+    public static final String CURRENCY_IDENTIFIER =
+            "https://bluebutton.cms.gov/resources/codesystem/identifier-currency";
+    public static final String CURRENT = "current";
+    public static final String HISTORIC = "historic";
```

Comment:
Rename to something like:
CURRENT_MBI
HISTORIC_MBI

to distinguish.

CURRENT also conflicts with what is in Enumerations.java.  Perhaps those definitions can be of use.